### PR TITLE
Add PPP and JUnit

### PR DESCRIPTION
**Pull request by mrwsy1**
regarding #192 
increase test coverage from 64% to 82%

### CODE REVIEW

File: docs/team/shuyi.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    + Provides the user with a list of faculties in NUS.
+
+* List routes of all buses: `/allbus`
+    + Provides the user with a list containing the routes of all shuttle buses available in NUS.
+
+* List all favorite commands: `/listfav`
+    + Allows the user to see a list of all his favorite commands with indexes and descriptions.
+
+* Clear all favorite commands: `/clearfav`
+    + Allows the user to clear his list of favorite commands.
```
Comment (yuxinwu99) [inline docs/team/shuyi.md]: Maybe you can justify why these feature are added.

File: src/test/java/seedu/duke/logic/commands/dinecommand/DineCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    }
+
+    @Test
+    public void executeCommand_validInput_expectNoException() {
+        String input = "science";
+        DineCommand com = new DineCommand(input);
+        assertDoesNotThrow(com::executeCommand);
+    }
+
+}
```
Comment (yuxinwu99) [inline src/test/java/seedu/duke/logic/commands/dinecommand/DineCommandTest.java]: Github suggests for you to end with a new line.
```suggestion
}

```

File: src/test/java/seedu/duke/logic/commands/dinecommand/DineInfoCommandTest.java [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    }
+
+    @Test
+    public void executeCommand_validInput_expectNoException() {
+        String input = "arise";
+        DineInfoCommand com = new DineInfoCommand(input);
+        assertDoesNotThrow(com::executeCommand);
+    }
+
+}
```
Comment (yuxinwu99) [inline src/test/java/seedu/duke/logic/commands/dinecommand/DineInfoCommandTest.java]:
```suggestion
}

```

File: docs/team/shuyi.md (hunk 1 above)
Comment (mrwsy1) [reply docs/team/shuyi.md]: noted.

**Review by yuxinwu99 — APPROVED** [review]

**Review by wamikamalik — APPROVED** [review]
LGTM!

**Pull request merged**